srcpkgs/python3-finvizfinance/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-finvizfinance'
+pkgname=python3-finvizfinance
+version=0.14.3
+revision=1
+wrksrc="finvizfinance-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-lxml python3-BeautifulSoup4 python3-requests python3-pandas python3-Sphinx python3-sphinx_rtd_theme"
+makedepends="python3-BeautifulSoup4"
+depends="python3-requests python3-BeautifulSoup4 python3-pandas python3-lxml python3-dateutil"
```

Comment:
`python3-dateutil` isn't needed after all, the package just uses the python3 built-in `datetime` module